lib/matplotlib/backend_bases.py

```diff
@@ -3551,7 +3593,8 @@ def _key_press(self, event):
         name = self._keys.get(event.key, None)
         if name is None:
             return
-        self.tool_trigger_event(name, canvasevent=event)
+        try: self.tool_trigger_event(name, canvasevent=event)
```

Comment:
Good question, I didn't know if it needed anything, should I send a do a ``warning.warn``, and/or send a message to the toolbar?

Thinking of which, what happens if the toolbar doesn't exist, what happens to the messages... if no toolbar we coul hook up a listener to the messages, and pass a severity level as part of the event, low severity levels pass straight through, higher ones will give go through ``warning.warn``.